test/uk/gov/hmrc/singlecustomeraccountwrapperdata/connectors/MessageConnectorSpec.scala

```diff
@@ -114,8 +122,34 @@ class MessageConnectorSpec extends AsyncWordSpec with Matchers with WireMockHelp
 
     }
   }
+
+  "The messageConnector with default settings" must {
+    "trigger a timeout if the request for get unread message count takes more than 1 second" in {
+
+      server.stubFor(
+        get(anyUrl()).willReturn(
+          aResponse()
+            .withStatus(OK)
+            .withBody("You haven't seen me, right!")
+            .withFixedDelay(2000)
+        )
+      )
+
+
+        implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(3, Seconds))
+         val position: Position = Position("MessageConnector.scala","uk/gov/hmrc/singlecustomeraccountwrapperdata/connectors/MessageConnector.scala",31)
+
+        val SUT: MessageConnector = injector.instanceOf[MessageConnector]
+
+        val result = SUT.getUnreadMessageCount(scala.concurrent.ExecutionContext.global, HeaderCarrier())
+
+        result.isReadyWithin(1 second) mustBe false
```

Comment:
It's false because the delay is 2 seconds and the result within 1 second is `Future(<not completed>)`

I have replace the patienceConfig with ` whenReady(result) { res =>
        res mustBe None
      }  `
      
      and it is passing, so I will remove the unnecessary code.
      
      Thanks!